canvas_grab/__main__.py

```diff
@@ -79,7 +89,7 @@ def greeting():
         except KeyboardInterrupt:
             print("User canceled the configuration process")
             return
-        Path(args.config_file).write_text(
+        Path(config_file).write_text(
```

Comment:
Seems that `config_file` itself is a Path object?

Reply:
Forgot this 😅